velox/common/base/Exceptions.h

```diff
@@ -196,6 +196,17 @@ DECLARE_CHECK_FAIL_TEMPLATES(::facebook::velox::VeloxRuntimeError);
       /* isRetriable */ false,                                      \
       ##__VA_ARGS__)
 
+#define VELOX_CHECK_FUZZER_ALLOWS(expr, ...)                          \
```

Comment:
Can we check the error message in fuzzer to tolerate specific exception. It might not be good to change production code for testing purpose? Thanks!